### PR TITLE
Communication verousca

### PR DESCRIPTION
**Pull request by Verousca**
## General Checks

- [ ] the branch is up to date with `main`/`master`
- [ ] the code works when pulled and run locally
- [ ] All CI checks pass
- [ ] all conflicts are resolved (if any)
- [ ] PR has a descriptive title
- [ ] PR has appropriate labels and milestones for easy identification
- [ ] PR it is assigned to the owner
- [ ] reviewers are assigned
- [ ] the PR contributes only one focused change
- [ ] It is in the appropriate column in the project board (if necessary)
- [ ] has short and clear description
- [ ] is linked to an issue (if it is related)
- [ ] feedback is addressed (if any and if it is appropriate feedback.)

## Markdown

- [ ] the markdown source is formatted
- [ ] spelling and grammar is correct in all text
- [ ] The markdown looks correct when you preview the file
- [ ] all links and images work

### CODE REVIEW

File: .cspell.json [hunk 1]
```diff
@@ -1,21 +1,22 @@
 {
```
Comment (barny-ar) [inline .cspell.json]: you dont need to add the .cspell.json file. so please remove it

File: planning/communication-plan.md [hunk 2]
```diff
@@ -28,21 +28,20 @@ how often will we get in touch on each channel, and what we will discuss there:
 
 ### Availability for calling/messaging
 
-| Day     |   Monday    |   Tuesday   |  Wednesday  |  Thursday   |   Friday    |  Saturday   |   Sunday   |
-| ------- | :---------: | :---------: | :---------: | :---------: | :---------: | :---------: | :--------: |
-| _Brian_ | 8am - 12pm  | 8am - 12pm  | 8am - 12pm  | 8am - 12pm  | 8am - 12pm  | 8am - 12pm  | 8am - 12pm |
-| _Inna_  | 15am - 12pm | 15am - 12pm | 15am - 12pm | 9am - 12pm  | 12am - 12pm | 9am - 12pm  |  --------  |
-| _Azima_ | 13am - 21pm | 13am - 21pm | 13am - 21pm | 13am - 21pm | 15pm - 18pm | 10am - 14pm |  --------  |
-| _name_  | 00am - 00pm | 00am - 00pm | 00am - 00pm | 00am - 00pm | 00pm - 00pm | 00am - 00pm |  --------  |
```
Comment (barny-ar) [inline planning/communication-plan.md:36]: please run formating commandes before pushing. 
- npm run format
- npm run format:check
- npm run lint:ls
- npm run lint:md
- npm run spell-check

File: .cspell.json (hunk 1 above)
Comment (Verousca) [reply .cspell.json]: Okay will do. Thank you.

File: planning/communication-plan.md (hunk 2 above)
Comment (Verousca) [reply planning/communication-plan.md:36]: I did run these but I'll try it again.

**Comment by barny-ar** [comment]
only push one file-which is the .md file. i see 3 file changes

**Pull request closed without merging**